src/scancode/cli.py

```diff
@@ -785,4 +776,4 @@ def save_results(scanners, only_findings, files_count, results, format, options,
                     raise e
         return
 
-    write_formatted_output(scanners, files_count, version, notice, results, format, options, input, output_file, echo_stderr)
+    write_formatted_output(files_count, version, notice, results, format, options, input, output_file, echo_stderr, scan_output_plugins)
```

Comment:
The simplest at this stage might be to remove anything that is not a hookspec for formats. You can add this later alright and this will make your code cleaner